docs/data/toolpad/getting-started/quickstart.md

```diff
@@ -0,0 +1,150 @@
+# Quickstart
+
+Get started building a Toolpad app!
+
+<p class="description">
+This page will guide you into creating your very first Toolpad application. We will create a "hello world" type of application that will ask for a name and print a greeting.
+</p>
+
+## Prerequisites
+
+Make sure you have a running Toolpad instance.
+
+1. Download the docker compose file
+
+   ```sh
+   curl -LO https://raw.githubusercontent.com/mui/mui-toolpad/master/docker/compose/docker-compose.yml
+   ```
+
+1. Start the docker compose services
+
+   ```sh
+   docker-compose -f docker/compose/docker-compose.yml up -d
+   ```
+
+Toolpad will be accessible under `http://localhost:3000/`.
+
+## Navigating Toolpad
```

Comment:
Does it make sense to make this part of the quickstart tutorial? 
If I understand correctly, the reason it's put here is so that we can refer to the sections when explaining how to do an action. Maybe it's better to just make the screenshots clearer? Perhaps with arrows? In a later stage maybe even animating them?

Reply:
> If I understand correctly, the reason it's put here is so that we can refer to the sections when explaining how to do an action

That's exactly the reason why I put it there

> Maybe it's better to just make the screenshots clearer? Perhaps with arrows?

Do you mean like don't use overlays with text but put something like rectangular shapes and point to the them with arrow and write a name of a section outside of the screenshot?

Also I was thinking that maybe @gerdadesign could take over this and make it "right", I just wanted to do a first pass to have something to use 😅

Reply:
> That's exactly the reason why I put it there

I think there could be value in making our quickstart easy enough to follow without needing to run the reader through a glossary of terms. We can do that by, instead of describing it, making the screenshots in the steps indicate what is to be clicked.

Reply:
I'd prefer to "teach" users terminology and then use text to refer to the actual section. While screenshots might be more explicit, they are also harder to maintain, so if that makes it possible to minimise amount of screenshots - I'd suggest to go that way

Reply:
Yep, I think that makes sense for general documentation. But we're talking about a quickstart tutorial here. I feel like the primary goal of this section would be to get users as quickly as possible to that dopamine hit of having built a working application. It will give them an overall feel of what it's like building an application, as well as a starting point to go tinkering with it on their own. I'm wondering whether too much theory in here could put off users finishing the tutorial.

Reply:
You are right 👍 I moved it to `Overview` (previously introduction) section